### DISCUSSION TITLE
Confirmation behavior

### DISCUSSION THREAD

**Discussion by cassepipe** [LunarVim/LunarVim#4059] [Q&A] [answered]
I set `vim.opt.confirm = true` but it does not behave like in vim:

Lunarvim, instead of asking me if I want to save the currently unsaved buffer, asks me if I am sure I want to quit.

Which means I have to type `n`, then save, then leave
Instead of just typing `y` to save or `n` to quit without saving.

I suspect `lunarvim` is taking over the native behavior and use a different dialog

Do you know how I can disable Lunarvim's dialog to get the native one ?

**Comment by cassepipe** [comment]
Even better, couldn't Lunarvim's current behavior be removed in favor or the native confirm  option being on by default ?

**Reply by LostInTheLogs** [reply]
you can remap leader c in whichkey

**Reply by cassepipe** [reply]
What would I remap it to ?

**Reply by LostInTheLogs** [reply]
`:q` `:bd`, a plugin, whatever you want

**Reply by cassepipe** [reply]
It works indeed. I thought it was already mapped to `:bd` that's why I asked.  Any reason what this is not the defaut ? Whatt is it currently mapped to ?

**Reply by LostInTheLogs** [reply]
our own function that preserves the layout of windows

**Comment by LostInTheLogs** [comment]
what do you use to quit?

**Reply by cassepipe** [reply]
It generally happens when I want to close a buffer with `<leader>c` and I get :
`No write since last change for $FILE. Close it anyway ? [y]es or [n]o  (default: no):`

**Reply by LostInTheLogs** [reply]
we fixed that for leader q, and forgot leader c

**Comment by LostInTheLogs** [comment, marked as answer]
https://github.com/LunarVim/LunarVim/issues/4172

**Reply by cassepipe** [reply]
Thanks !
